services/cadip/rs_server_cadip/api/cadip_search.py

```diff
@@ -85,13 +85,27 @@ def __init__(self, request: Request | None = None, readwrite: Literal["r", "w"]
     @handle_exceptions
     async def process_search(self, collection: dict, odata_params: dict) -> stac_pydantic.ItemCollection:
         """Do the search for the given collection and OData parameters."""
+        sortby = self.request.query_params.get("sortby", None)
```

Comment:
if "sortby" is missing from the arguments, it will take None, and its value will be None also in the process_session_search function below. As for adgs, maybe it would be better if it took the default value "-datetime" here, and be a mandatory argument in the process_session_search function below.

Reply:
Removed from signature, default is now - published (PublicationDate desc)